mapstruct-lombok/build.gradle

```diff
@@ -1,6 +1,5 @@
 plugins {
     id 'java'
-    id 'net.ltgt.apt' version '0.9'
```

Comment:
@gunnarmorling Multitasking 😀. There already exists mapstruct/mapstruct.org/issues/63. I updated it so we don't forget to add some info about this. It should be part of the IDE Support section